nbs/mint.json

```
@@ -37,6 +37,7 @@
         "examples/exogenous_variables.html",
         "examples/time_series_scaling.html",
         "examples/automatic_hyperparameter_tuning.html",
+        "examples/Cross_Validation_Tutorial.html",
```

These should be all lowercase. You can see what the name looks like [here](https://github.com/Nixtla/neuralforecast/blob/docs/examples/cross_validation_tutorial.html.mdx)
```suggestion
        "examples/cross_validation_tutorial.html",
```